hass_nabucasa/acme.py

```diff
@@ -113,16 +112,16 @@ def certificate_available(self) -> bool:
     @property
     def is_valid_certificate(self) -> bool:
         """Validate date of a certificate and return True is valid."""
-        if not self._x509:
+        if self.expire_date is None:
```

Comment:
I'd save the property return value in a walrus variable so we don't need to access the property more than once.
```suggestion
        if expire_date := self.expire_date is None:
```